cypress/e2e/deputy_details.cy.js

```diff
@@ -123,4 +123,31 @@ describe("Deputy details tab", () => {
             ).should("contain", "some info for the pa deputy");
         });
     });
+
+    describe("Delete a deputy", () => {
+        it("can delete a deputy", () => {
+            cy.setCookie("success-route", "/deputies/1");
+            cy.get('[href="/supervision/deputies/1/delete-deputy"]').click();
```

Comment:
Cypress best practice is to avoid using selectors and attributes for selecting elements because they are very brittle and do not reflect actual use in test scenarios. For example, if someone copied the delete button to create a new one but forgot to update the href, this test would start failing, despite the delete functionality not being affected.

Buttons should be addressed either by their text or a `data-cy` attribute.

Reply:
done